sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -275,7 +275,7 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
       });
     }
     return assessmentHistoryQuery
-      .orderBy("assessment.studentAssessmentStatus", "DESC")
+      .orderBy("assessment.studentAssessmentStatus", "ASC")
```

Comment:
Please remove the `orderBy("assessment.studentAssessmentStatus", "ASC")` and keep only the `OrderBy("assessment.submittedDate", "DESC")` to comply with the business AC.